cargo-process.el

```diff
@@ -150,10 +150,11 @@
 (defun cargo-process--start (name command)
   "Start the Cargo process NAME with the cargo command COMMAND."
   (let* ((buffer (concat "*Cargo " name "*"))
+         (project-root (cargo-process--project-root))
          (path cargo-process--custom-path-to-bin)
          (path (and path (file-name-as-directory path)))
-         (command (cargo-process--maybe-read-command (concat path command)))
-         (project-root (cargo-process--project-root)))
+         (command (cargo-process--maybe-read-command
+                   (concat "cd " project-root " && " path command))))
```

Comment:
The way to go would definitely be to adjust `default-directory` in scope.
However I'd write it like this: `(default-directory (or project-root default-directory))`
Which is equal to the if statement, but I try to avoid those.

Anyway I'll make